package_managers/download_pkgs.bzl

```diff
@@ -82,7 +83,7 @@ Args:
   packages: list of packages to download. e.g. ['curl', 'netbase']
 """
 
-def download_image_pkgs(name, base, packages=[]):
+def download_image_pkgs(name, base, packages=[], additional_repos=[]):
```

Comment:
Can you add a test to tests/package_manager/BUILD with additional_repos and verify the tar has the source.list file in download_pkgs_run_test.sh ?

It will make things easier for me in my PR which removes the download_image_packages macro